app/grandchallenge/algorithms/templates/algorithms/job_detail.html

```diff
@@ -0,0 +1,258 @@
+                    <i class="fa fa-eye"></i> Open Result in Viewer
+                </a>
+            {% endif %}
+
+            <dl class="row">
+                <dt class="col-sm-3">Result ID</dt>
+                <dd class="col-sm-9">{{ object.pk }}</dd>
+
+                <dt class="col-sm-3">Algorithm</dt>
+                <dd class="col-sm-9"><a
+                        href="{{ object.algorithm_image.algorithm.get_absolute_url }}">{{ object.algorithm_image.algorithm }}</a>
+                </dd>
+
+                <dt class="col-sm-3">Algorithm Version</dt>
+                <dd class="col-sm-9">{{ object.algorithm_image.pk }}</dd>
+
+                <dt class="col-sm-3">Creator</dt>
+                <dd class="col-sm-9">{{ object.creator|user_profile_link }}</dd>
+
+                <dt class="col-sm-3">Created</dt>
+                <dd class="col-sm-9">{{ object.created }}</dd>
+
+                <dt class="col-sm-3">Status</dt>
+                <dd class="col-sm-9">
+                    <span class="badge
+                        {% if object.status == object.FAILURE or object.status == object.CANCELLED %}
+                            badge-danger
+                        {% elif object.status == object.RETRY %}
+                            badge-warning
+                        {% elif object.status == object.SUCCESS %}
+                            {% if object.stderr %}
+                                badge-warning
+                            {% else %}
+                                badge-success
+                            {% endif %}
+                        {% else %}
+                            badge-info
+                        {% endif %}">
+                        {{ object.get_status_display }}{% if object.status == object.SUCCESS and object.stderr %}, with
+                            Warnings{% endif %}
+                    </span>
+                </dd>
+
+                {% if object.error_message %}
+                    <dt class="col-sm-3">Error Message</dt>
+                    <dd class="col-sm-9">{{ object.error_message }}</dd>
+                {% endif %}
+
+                {% if object.duration %}
+                    <dt class="col-sm-3">Duration</dt>
+                    <dd class="col-sm-9">{{ object.duration|naturaldelta }}</dd>
+                {% endif %}
+
+                {% if object.comment %}
+                    <dt class="col-sm-3">Comment</dt>
+                    <dd class="col-sm-9">{{ object.comment }}</dd>
+                {% endif %}
+
+                <dt class="col-sm-3">Visibility</dt>
+                <dd class="col-sm-9">
+                    {% if object.public %}
+                        <i class="fa fa-eye text-success"></i> Result and images are public
+                    {% else %}
+                        {% if object.viewers.user_set.all|length > 1 %}
+                            {# TODO: Hack, we need to exclude the creator rather than checking the length is > 1 #}
```

Comment:
Can't the algorithm's editors view all results as well?

Reply:
This status is about members of the viewers group, on the Users tab there is a list of all the other groups that have access via the `viewer_groups` many to many, which could be empty, the editors, and/or archive groups (PR incoming from Harm).

Reply:
ok